Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracksPrompt_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+ALCARECOPPSCalMaxTracksFilterPrompt = hlt.hltHighLevel.clone(
+  TriggerResultsTag = cms.InputTag("TriggerResults","","HLTX"),
+  HLTPaths = ['HLT_PPSMaxTracksPerRP4_v1']
+)
+
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemTriggerRawToDigi as _totemTriggerRawToDigi
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemRPRawToDigi as _totemRPRawToDigi
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import ctppsDiamondRawToDigi as _ctppsDiamondRawToDigi
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import totemTimingRawToDigi as _totemTimingRawToDigi
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import ctppsPixelDigis as _ctppsPixelDigis
+
+totemTriggerRawToDigiAlCaRecoProducerPrompt = _totemTriggerRawToDigi.clone(rawDataTag = cms.InputTag('hltPPSCalibrationRaw'))
```

Comment:
Please remove also here the explicit types for the cloned parameters

Reply:
At later stage done in https://github.com/cms-sw/cmssw/pull/36273/commits/18c83a8f88a9ba2168b0438515eb0d211ecdd53c